### DISCUSSION TITLE
Plot for sale prompt not showing up?

### DISCUSSION THREAD

**Discussion by tate98** [TownyAdvanced/Towny#4633] [Q&A] [answered]
So we have a town setup and have claimed plots and put them up for sale, but shouldn't there be a prompt in chat when a player walks into a plot that is for sale? Right now there is nothing. I might be doing something completely wrong but if anyone knows I would appreciate the help.

**Comment by LlmDl** [comment, marked as answer]
Plots' chunk notifications only show up if the plot you've just left did not have the same setting.

For example: going from a plot with PVP off to a plot with PVP on, you'll see PVP in the chunk notifications.

If your plots are all the same price and there's no not-for-sale plots in between them, you'll likely not see the for-sale because it did not change.
